orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/support/TargetServerGroupResolver.groovy

```diff
@@ -168,7 +168,14 @@ class TargetServerGroupResolver {
           throw RetrofitError.conversionError(response.url, response, converter, responseType, ce)
         }
       } catch (RetrofitError re) {
-        if (re.kind == RetrofitError.Kind.NETWORK && i < maxRetries) {
+        if (i > maxRetries) {
+          log.error("Max retries exhausted", re)
```

Comment:
How can it get there if `i > maxRetries`? The loop condition at line 154 will prevent it, surely.